src/main/java/au/com/centrumsystems/hudson/plugin/buildpipeline/BuildPipelineView.java

```diff
@@ -767,16 +767,13 @@ public Item doCreateItem(final StaplerRequest req, final StaplerResponse rsp) th
     public boolean hasPermission(final Permission p) {
         try {
             boolean display = true;
-            //tester la liste vide seulement en lecture
-            if (READ.name.equals(p.name)) {
+            if (READ.equals(p)) {
                 if (isEmpty()) {
-                    display = false;
+                    display = super.hasPermission(View.CONFIGURE);

```

Comment:
Right it does in [hudson.security.AuthorizationStrategy](https://github.com/jenkinsci/jenkins/blob/master/core/src/main/java/hudson/security/AuthorizationStrategy.java#L102-L104).

The behavior is slightly different from what the Build Pipeline plugin currently does and I thought we wanted to keep that behavior. 

Currently, if a user has `View.READ` permissions, it wouldn't see empty views in the tabs.
With the ACL behavior, if a user has `View.READ` permissions, it can see empty views in the tabs.